client/styles/components/_editor.scss

```diff
@@ -139,6 +141,10 @@ pre.CodeMirror-line {
 .CodeMirror-find-controls {
   display: flex;
   align-items: center;
+
+  @media (max-width: 580px) {
+    padding: 0.6rem 0rem;
+  }
```

Comment:
This seems unrelated to the `z-index` issues.  So let's save that for a PR where we fix the layout.

Reply:
OK, I will raise a new pr for it. Thank you